### PR TITLE
Socket stuff with multithreading

### PR DESCRIPTION
**Pull request by dariusau**
Don't accept the pull request just yet unless you want to test it.
I haven't tested it thoroughly enough.
Or added enough documentation yet.
FYI, the output.csv file produced from this is on the computer now (not the robot), but its still corrupted.

### CODE REVIEW

**Review by dariusau — COMMENTED** [review]
Oh and this code is messy mind you. You'll have to clean it up... again lol

File: SocketServer.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    mySocket.bind((host, port))
+
+    mySocket.listen(1)
+    global conn
+    conn, addr = mySocket.accept()
+
+    # Setting up the .csv file
+    print("Opening output.csv")
+    writer = openCSVFile()
+    writer.writeHeader()
```
Comment (dariusau) [inline SocketServer.py:73]: Need to check if the file already exists or not. If it does already exist with previous values, new values (as well as a new header row) are appended to the file

Comment (dariusau) [reply SocketServer.py:73]: Alright did that in commit 83def8e "Added functionality to clear file before writing to .csv file... "
You can change that if you want, its in the writeCSV file (which I was reluctant to change, but oh well)

**Comment by ministiy** [comment]
I swear I will merge this request one of these days

**Pull request merged**